relay-server/src/services/stats.rs

```diff
@@ -21,6 +22,7 @@ pub struct RelayStats {
     upstream_relay: Addr<UpstreamRelay>,
     #[cfg(feature = "processing")]
     redis_pools: Option<RedisPools>,
+    async_pools_metrics: Vec<AsyncPoolMetrics>,
```

Comment:
We could hardcode just the two pools, but I wanted to make it more generic so that no changes would be required if we were to add another pool.